pyaedt/generic/settings.py

```diff
@@ -0,0 +1,528 @@
+    def enable_local_log_file(self):
+        """Enable/Disable the local pyaedt log file logging in projectname.pyaedt project folder. Default is `True`.
+
+        Returns
+        -------
+        bool
+        """
+        return self._enable_local_log_file
+
+    @enable_local_log_file.setter
+    def enable_local_log_file(self, value):
+        self._enable_local_log_file = value
+
+    @property
+    def global_log_file_name(self):
+        """Get/Set the global pyaedt log file path. Default is pyaedt_username.log.
+
+        Returns
+        -------
+        str
+        """
+        return self._global_log_file_name
+
+    @global_log_file_name.setter
+    def global_log_file_name(self, value):
+        self._global_log_file_name = value
+
+    @property
+    def enable_pandas_output(self):
+        """
+        Set/Get a flag to use Pandas to export dict and lists. This applies to Solution data output.
+        If ``True`` the property or method will return a pandas object in CPython environment.
+        Default is ``False``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._enable_pandas_output
+
+    @enable_pandas_output.setter
+    def enable_pandas_output(self, val):
+        self._enable_pandas_output = val
+
+    @property
+    def enable_debug_methods_argument_logger(self):
+        """
+        Set/Get a flag to plot methods argument in debug logger.
+        Default is ``False``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._enable_debug_methods_argument_logger
+
+    @enable_debug_methods_argument_logger.setter
+    def enable_debug_methods_argument_logger(self, val):
+        self._enable_debug_methods_argument_logger = val
+
+    @property
+    def force_error_on_missing_project(self):
+        """Set/Get a flag to check project path.
+        If ``True`` when passing a project path, the project has to exist otherwise it will raise an error.
+        Default is ``False``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._force_error_on_missing_project
+
+    @force_error_on_missing_project.setter
+    def force_error_on_missing_project(self, val):
+        self._force_error_on_missing_project = val
+
+    @property
+    def disable_bounding_box_sat(self):
+        """Set/Get Bounding Box Sat enablement.
+
+        Returns
+        -------
+        bool
+        """
+        return self._disable_bounding_box_sat
+
+    @disable_bounding_box_sat.setter
+    def disable_bounding_box_sat(self, val):
+        self._disable_bounding_box_sat = val
+
+    @property
+    def use_grpc_api(self):
+        """Set/Get GRPC API usage or Legacy COM Object. This is the user setting.
+
+        Returns
+        -------
+        bool
+        """
+        return self._use_grpc_api
+
+    @use_grpc_api.setter
+    def use_grpc_api(self, val):
+        """Set/Get GRPC API usage or Legacy COM Object. This is the user setting."""
```

Comment:
```suggestion
        """Set/Get GRPC API usage or Legacy COM Object. This is the user setting."""
```
I don't think we need to a docstring here. According to Alex K., attributes should be described as a noun. If someone needs to know how to set or get the attribute, then you would add an ``Examples`` section.